### PR TITLE
Image create sem 1

### PR DESCRIPTION
**Pull request by JacksonM8**

### CODE REVIEW

File: docker.h [hunk 1]
```diff
@@ -22,7 +22,7 @@ std::string param( const std::string& param_name, bool param_value);
 std::string param( const std::string& param_name, int param_value);
 std::string param( const std::string& param_name, JSON_DOCUMENT& param_value);
 
-std::string jsonToString(JSON_VALUE & doc);
+std::string jsonToString(const JSON_VALUE& doc);
```
Comment (mitchell-conrad) [inline docker.h:25]: Nice

File: docker.cpp [hunk 2]
```diff
@@ -268,7 +284,7 @@ JSON_DOCUMENT Docker::requestAndParse(Method method, const std::string& path, lo
 
         doc.AddMember("success", false, doc.GetAllocator());
         // Forced bool parameter required to prevent ambiguous function call (long appears to cause problems)
-        doc.AddMember("code", (bool)status, doc.GetAllocator());
+        doc.AddMember("code", (int)status, doc.GetAllocator());
```
Comment (mitchell-conrad) [inline docker.cpp]: This comment is now at odds with the code.

**Review by mitchell-conrad — APPROVED** [review]

File: docker.h (hunk 1 above)
Comment (JacksonM8) [reply docker.h:25]: 👍

File: docker.cpp (hunk 2 above)
Comment (JacksonM8) [reply docker.cpp]: Thanks, will fix

**Review by mitchell-conrad — APPROVED** [review]

**Pull request merged**